client/src/Views/Landing/Landing.js

```diff
@@ -5,17 +5,18 @@ import { Route, Switch } from "react-router-dom";
 import Navbar from "./LandingNavbar";
 import Login from "./Login";
 import Register from "./Register";
+import ProcessGmailToken from "./ProcessGmailToken";
 
 const styles = () => ({
   root: {
     height: "100vh",
     width: "100%",
     backgroundColor: "#F4F6FC",
-    overflow: "auto"
-  }
+    overflow: "auto",
+  },
 });
 
-const Landing = props => (
+const Landing = (props) => (
```

Comment:
looks like we have different settings, @Disquietude ?

Reply:
This should be fixed now